packages/web-components/src/components/locale-modal/__stories__/README.stories.mdx

```diff
@@ -0,0 +1,111 @@
+import { Preview, Props, Description, Story } from '@storybook/addon-docs/blocks';
+import contributing from '../../../../../../docs/contributing-license.md';
+import corsproxy from '../../../../../../docs/cors-proxy.md';
+
+# Locale modal
+
+> The locale modal allows users to change geographic regions and translate pages
+> to those region languages, if available.
+
+<Preview withToolbar>
+  <Story id="components-locale-modal--default" height="600px" />
+</Preview>
+
+> 💡 Check our
+> [CodeSandbox](https://githubbox.com/carbon-design-system/ibm-dotcom-library/tree/master/packages/web-components/examples/codesandbox/components/locale-modal)
+> example implementation.
+
+## Getting started
+
+##### JS
+
+```javascript
+import '@carbon/ibmdotcom-web-components/es/components/locale-modal/locale-modal';
+import '@carbon/ibmdotcom-web-components/es/components/locale-modal/regions';
+import '@carbon/ibmdotcom-web-components/es/components/locale-modal/region-item';
+import '@carbon/ibmdotcom-web-components/es/components/locale-modal/locale-search';
+import '@carbon/ibmdotcom-web-components/es/components/locale-modal/locale-item';
+```
+
+##### HTML
+
+```html
+<dds-locale-modal lang-display="United States — English" open>
+  <dds-regions>
+    <dds-region-item name="Americas"></dds-region-item>
```

Comment:
Would adopters have to manage this themselves? Right now this is all driven by the alternative links in the `<head/>`, since each page authored can vary on available locale/langauges. This is also existing logic from v18 so that it continues to be managed that way on the page level.

Reply:
@jeffchew No, the plan is creating a Redux store that allows our users (or our code) to auto-generate `<dds-region-item>` and `<dds-locale-item>`. The pre-redux version of the code can be seen at: https://github.com/carbon-design-system/ibm-dotcom-library/blob/b91fb76d76a126557ba9f26c93e7ade72d0a82c0/packages/web-components/src/components/locale-modal/locale-modal-container.ts

Reply:
Ok, in this case we might need a `simplified` getting started documentation on what the user is actually implementing, and how the data is getting loaded in. For at least the React version, there is just the addition of `<LocaleModal />` into where it needs to go, and behind the scenes it makes the API call and gets hydrated accordingly. We want to make sure that adopting this (and web components in general) is as easy as possible for adopters. We can certainly have extended documentation on the inner workings of this and how it can be modified for usage, but more for `advanced` users.

Reply:
OK thank you for your insights, let me update the doc once I get the updated version of `<dds-locale-modal-container>` with #3197 incorporated.